src/components/Layout/LayoutHeader.tsx

```diff
@@ -22,13 +22,13 @@ import { usePathWithoutHash } from '@/utils/usePathWithoutHash';
 
 export const LayoutHeader = ({
   currentPlatform,
-  isGen2,
+  isGen1,
   pageType = 'inner',
   showLastUpdatedDate = true,
   showTOC
 }: {
-  currentPlatform?: Platform | undefined;
```

Comment:
Not optional anymore since all pages should have a platform now

Reply:
Does this checkout for the Contribute page?

Reply:
I think so. At the moment the contribute site doesn't accept a platform. On the production site it seems to work similar to my staging site. When we're on a platform other than react, and we navigate to contribute, it just takes us to the contribute page, and the platform navigator shows react